xcode/Subconscious/Shared/UIKit/BlockEditor/BlockEditorModel.swift

```diff
@@ -113,17 +119,20 @@ extension BlockEditor {
             autofocus: Bool = false
         )
         /// Load related notes (backlinks)
-        case fetchRelated
-        case succeedFetchRelated(_ related: [EntryStub])
-        case failFetchRelated(_ error: String)
-        case refreshTranscludes
-        case succeedRefreshTranscludes([EntryStub])
-        case fetchTranscludesFor(id: UUID, slashlinks: [Slashlink])
-        case succeedFetchTranscludesFor(
+        case loadRelated
+        case succeedLoadRelated(_ related: [EntryStub])
+        case failLoadRelated(_ error: String)
+        /// Load transcludes into cache for entire document
+        case loadTranscludes
+        case succeedLoadTranscludes([EntryStub])
+        case loadTranscludesFor(id: UUID, slashlinks: [Slashlink])
+        case succeedLoadTranscludesFor(
             id: UUID,
-            fetched: [EntryStub]
+            transcludes: [EntryStub]
         )
-        case failFetchTranscludesFor(id: UUID, error: String)
+        case failLoadTranscludesFor(id: UUID, error: String)
```

Comment:
Very nice! This is exactly the flexibility I think we need to minimise perceived load time.